calico/felix/actor.py

```diff
@@ -232,11 +249,28 @@ def _step(self):
                 except BaseException as e:
                     _log.exception("Exception processing %s", msg)
                     results.append(ResultOrExc(None, e))
+                    message_exception = e.__class__.__name__ +\
+                                                      ': ' + str(e)
                     _stats.increment("Messages executed with exception")
                 else:
+                    message_exception = None
                     results.append(ResultOrExc(result, None))
                     _stats.increment("Messages executed OK")
                 finally:
+                    batch_uuids.add(msg.uuid)
+                    # Log the message info
+                    # Strings are enclosed in quotes. Numbers aren't.
+                    _message_log.info('"{uuid}": {data}'.format(
+                        uuid=msg.uuid,
+                        data=json.dumps({
+                            "received-time": int(time.time() * 1000),
```

Comment:
Why store milliseconds instead of just leaving the timestamp as seconds?  Presumably you'll be parsing this with Python, so you'll just have to convert it back again.